### PR TITLE
UPDATE: whist core version

### PR DESCRIPTION
**Pull request by Segelzwerg**
Requires https://github.com/Whist-Team/Whist-Core/pull/332

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -23,14 +23,16 @@ whist-server = "whist_server.cli:main"
 
 [tool.poetry.dependencies]
```
Comment (iTitus) [inline pyproject.toml:24]: please use `^major.minor` syntax and use `[tool.poetry.group.dev.dependencies]` for all dev dependencies acccording to https://python-poetry.org/docs/managing-dependencies/#dependency-groups

Don't forget to do `poetry update` and `poetry install` as well ;)

**Review by iTitus — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (Segelzwerg) [reply pyproject.toml:24]: Done

File: pyproject.toml [hunk 2]
```diff
@@ -23,27 +23,25 @@ whist-server = "whist_server.cli:main"
...
 PyJWT = "^2.7"
 splunk-sdk = "^1.7"
 bcrypt = "^4.0"
-whist-core = "^0.9.0"
+whist-core = "^0.9"
 python-multipart = "^0.0.6"
-uvicorn = "^0.23.1"
+uvicorn = "^0.23"
 
 [tool.poetry.dev-dependencies]
```
Comment (iTitus) [inline pyproject.toml]:
```suggestion
[tool.poetry.group.dev.dependencies]
```

**Review by iTitus — CHANGES_REQUESTED** [review]

**Comment by Segelzwerg** [comment]
@iTitus I think your comment was done on outdated version.

**Review by iTitus — APPROVED** [review]

**Pull request merged**